### PR TITLE
[DOC] Add new documentation about Query Workbench

### LINKED ISSUE #2156: [DOC] Consolidated Dev Tools
**What do you want to do?**
 
- [X] Request a change to existing documentation
- [X] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 Provide support for a consolidated development tooling experience. The work will require adding stubs where existing functionality is migrating from as well as improvements made to the tooling once inside. The scope is still under discussion as to what will fit into what milestone.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3033

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Add new documentation about query workbench

### Issues Resolved
#2156 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/query-workbench.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+{"index":{"_id":"6"}}
+{"account_number":6,"balance":5686,"firstname":"Hattie","lastname":"Bond","age":36,"gender":"M","address":"671 Bristol Street","employer":"Netagy","email":"hattiebond@netagy.com","city":"Dante","state":"TN"}
+{"index":{"_id":"13"}}
+{"account_number":13,"balance":32838,"firstname":"Nanette","lastname":"Bates","age":28,"gender":"F","address":"789 Madison Street","employer":"Quility","email":"nanettebates@quility.com","city":"Nogal","state":"VA"}
+{"index":{"_id":"18"}}
+{"account_number":18,"balance":4180,"firstname":"Dale","lastname":"Adams","age":33,"gender":"M","address":"467 Hutchinson Court","email":"daleadams@boink.com","city":"Orick","state":"MD"}
+```
+{% include copy-curl.html %}
+
+The following response confirms the data is indexed:
```
Comment (vagimeli) [inline _dashboards/query-workbench.md:36]: Is there a button we can add to collapse/expand the response?

File: _dashboards/query-workbench.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    - Select the **Clear** button to clear the query editor and run a new query. 
+
+5. Examine how the query is processed.
+    - Select the **Explain** button to examine how OpenSearch processes the query, including the steps involved and order of operations. 
+
+    The following image shows the explanation of the SQL query that was run in step 2.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-explain.png" alt="Query Workbench SQL query explanation pane" width="500">
+
+### Viewing indexes
```
Comment (vagimeli) [inline _dashboards/query-workbench.md:158]: Is there an option to save SQL and PPL queries?

**Comment by vagimeli** [comment]
@brijos Please review the Query Workbench draft content. I rewrote the [currently published content](https://opensearch.org/docs/latest/search-plugins/sql/workbench/) and expanded on the tutorial. Who else should I tag for technical review? Thanks!

File: _search-plugins/sql/ppl/index.md [hunk 3]
```diff
@@ -14,53 +14,16 @@ redirect_from:
...
-
-#### Example response
-
-firstname | lastname |
-:--- | :--- |
-Amber       | Duke
-Hattie      | Bond
-Nanette     | Bates
-Dale        | Adams
+## Next steps
```
Comment (vagimeli) [inline _search-plugins/sql/ppl/index.md:27]: SME: We need to include descriptions of each command. Please provide that information.

File: _dashboards/query-workbench.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+FROM <index_name>
+WHERE <field_name> = <field_value>
+```
+{% include copy.html %}
+
+In this syntax, replace `<index_name` with the name of the index where the document is stored, `<field_name` with the name of the field you want to search, and `<field_value` with the value you want to search for. The `*` in the `SELECT` statement means that you want to retrieve all fields from the document. If you only want to retrieve specific fields, replace `*` with a comma-separated list of field names. 
+
+### Deleting data
+
+To delete a document from an index, run a `DELETE` statement with the `WHERE` clause. The following is the basic SQL syntax to delete a document from an index:
```
Comment (kolchfa-aws) [inline _dashboards/query-workbench.md:188]: Same as above. The WHERE clause can be more complicated. Maybe remove this section altogether and make the previous section parallel to PPL?

**Review by kolchfa-aws — COMMENTED** [review]
Nice flow. Left some comments.

File: _dashboards/query-workbench.md (hunk 4 above)
Comment (vagimeli) [reply _dashboards/query-workbench.md:188]: Deleted the SQL sections about listing, retrieving, and deleting. If a developer wants to include this moving forward, we can have them write that information from the dev POV.

**Comment by vagimeli** [comment]
> LGTM

Thank you for stepping in to do a tech review! Your reviews are the best and most thorough :)

**Pull request merged**